index.html

```diff
@@ -62,6 +62,157 @@ <h3>LET'S CONNECT</h3>
                 </ul>
             </div>
         </section>
+
+        <section class="canopy-card">
+            <section class="canopy-card-section">
+                <div class="snapshoot-portfolio">
+                    <img src="img/Snapshoot Portfolio (4).png" alt="First canopy card">
+                </div>
+                <div class="project-card">
+                    <h2>Tonic</h2>
+                    <h3>CANOPY &nbsp; <img src="img/Counter (1).png" alt="dot">&nbsp;<span> Back End Devb &nbsp; <img src="img/Counter (1).png" alt="dot"> &nbsp; 2015</span></h3>
+                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                    <ul>
+                        <li>HTML</li>
+                        <li>CSS</li>
+                        <li>Javascript</li>
+                    </ul>
+                    <button type="button">See project</button>
+                </div>
+            </section>
+
+            <section class="canopy-second-card-section">
+                <div class="snapshoot-portfolio">
+                    <img src="img/Snapshoot Portfolio (6).png" alt="First canopy card">
+                </div>
```

Comment:
- Kindly center the image correctly, the margin left and right must be the same so that it will make your design stick more to the figma design as required in the project check the comparison below :+1: Kindly do the same for all cards. 

## Your design 

![image](https://user-images.githubusercontent.com/95297251/235215033-8e5e88d3-0d25-43a5-a1cc-8422faec143b.png)

## Figma design 

![image](https://user-images.githubusercontent.com/95297251/235214901-3746df9e-2b12-448d-b10f-f9a7dfa79343.png)